eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
 <fg=cyan>Welcome to eZ Platform!</fg=cyan>
 
 <options=bold>Quick Install:</>
-(Assuming the CLI user you execute commands with below is same that extracted/installed the software)
-<comment>    $  export SYMFONY_ENV="prod"</comment>
+<comment>    $  export SYMFONY_ENV="dev"</comment>
 <comment>    $  php ${consoleDir}/console ezplatform:install <type></comment>
-<comment>    $  php ${consoleDir}/console assetic:dump</comment>
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- "ezplatform:install" has different installer <type>s depending on your install, see <fg=green>INSTALL.md</> or <fg=green>README.md</> for which one to use.
-- For development use you can enable full debugging by setting SYMFONY_ENV to "dev".
-- Last command will give you url to frontend of installation, add "/ez" to reach backend.
+- "ezplatform:install" has different installer <type>s depending on your install, run command with <comment>--help</comment> to see your options.
+- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software,
+  otherwise see install instructions for directory permissions.
+- Last command will give you url to frontend of installation, add "/admin" to reach backend.
```

Comment:
`- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.`

Reply:
Thanks, fixed in f87e2d7